cbor/src/test/java/com/fasterxml/jackson/dataformat/cbor/gen/GeneratorSimpleTest.java

```diff
@@ -1,17 +1,16 @@
 package com.fasterxml.jackson.dataformat.cbor.gen;
 
+import com.fasterxml.jackson.core.JsonGenerationException;
```

Comment:
Please do not re-order imports: existing ordering is as per style guide.

Reply:
Got it, order restored.